provisioning/templates-api/common.sh

```diff
@@ -1,7 +1,20 @@
 #!/usr/bin/env bash
 
-# CD to the script directory
-cd "$(dirname "$0")"
+# Prevent direct run of the script
+if [ "${BASH_SOURCE[0]}" -ef "$0" ]
+then
+    echo 'This script should not be executed directly, please run "deploy.sh" instead.'
+    exit 1
+fi
+
+# Required ENVs
+: ${RELEASE_ID?"Missing RELEASE_ID"}
+: ${KEBOOLA_STACK?"Missing KEBOOLA_STACK"}
+: ${HOSTNAME_SUFFIX?"Missing HOSTNAME_SUFFIX"}
+: ${TEMPLATES_API_REPOSITORY?"Missing TEMPLATES_API_REPOSITORY"}
+: ${TEMPLATES_API_IMAGE_TAG?"Missing TEMPLATES_API_IMAGE_TAG"}
+: ${TEMPLATES_API_REPLICAS?"Missing TEMPLATES_API_REPLICAS"}
+: ${TEMPLATES_API_ETCD_REPLICAS?"Missing TEMPLATES_API_ETCD_REPLICAS"}
```

Comment:
Definiciu ENVs som presunul do `common.sh`, kedze to je ten subor, ktory sa bude vacsinou upravovat.